conans/client/conf/__init__.py

```diff
@@ -101,7 +101,7 @@
         exception: [null, dwarf2, sjlj, seh]  # Windows MinGW
         cppstd: [null, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20, 23, gnu23]
     msvc:
-        version: [170, 180, 190, 191, 192, 193]
+        version: [170, 180, 190, 191, 192, 193, 194]
         update: [null, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
Are we not missing update values then if it can go higher than 10? Or am I not seeing something?

Reply:
Conan ``msvc.update`` is the compiler update, not the IDE update. In that sense it is just and added version number to ``193+update``, ``194+update``, and it cannot be 10. So IDE 17.10 (ide version 10, update 10) => msvc 1940 (version=194, update=0)

Reply:
Ohh, not the IDE one, makes sense now :)